package/tests/testthat/_snaps/stable/tables-ADA-adat01.md

```diff
@@ -5,22 +5,22 @@
-      Patient with no positive samples at baseline       71             68             0           0            139    
-      Incidence of Treatment Emergent ADAs                                                                             
-        Post-baseline evaluable patients                134            132             0           0            266    
-      Patient positive for Treatment Emergent ADA        0              0              0           0             0     
-      Treatment-induced ADA                              0              0              0           0             0     
-      Treatment-enhanced ADA                             0              0              0           0             0     
-      Patient negative for Treatment Emergent ADA        0              0              0           0             0     
-      Treatment unaffected                               0              0              0           0             0     
-      —————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+                                                       A: Drug X    C: Combination   <Missing>   B: Placebo   All Drug X 
+                                                        (N=134)        (N=132)         (N=0)      (N=134)       (N=266)  
+      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      Baseline Prevalence of ADAs                                                                                        
+        Baseline evaluable patients                       134            132             0           0            266    
+        Patient with a positive sample at baseline     63 (47.0%)     64 (48.5%)         0           0        127 (47.7%)
+        Patient with no positive samples at baseline       71             68             0           0            139    
+      Incidence of Treatment Emergent ADAs                                                                               
+        Post-baseline evaluable patients                  134            132             0           0            266    
+        Patient positive for Treatment Emergent ADA        0              0              0           0             0     
+          Treatment-induced ADA                            0              0              0           0             0     
+          Treatment-enhanced ADA                           0              0              0           0             0     
+        Patient negative for Treatment Emergent ADA        0              0              0           0             0     
+          Treatment unaffected                             0              0              0           0             0     
```

Comment:
I do not know if the change in indentation is fine. @khatril do you know if we need to align with the original indentation here?

Reply:
it seems correct to me anyway

Reply:
Yes, this looks correct to me